Since 2.19.0 builds fails due to ubiquitous language support

### I tried this:

Just a regular `eventcatalog build`. 
We don't use any of the ubiquitous language features yet (I assume thats why its undefined).

### This happened:

```
src/pages/docs/[type]/[id]/language/[dictionaryId]/index.astro
14:58:37 [ERROR] [build] Failed to call getStaticPaths for src/pages/docs/[type]/[id]/language/[dictionaryId]/index.astro
Cannot read properties of undefined (reading 'data')
```

Might be caused by https://github.com/event-catalog/eventcatalog/commit/5f8dbfe190a37108a37728b1e9c452093b672f13#diff-d0a041c1a6a12f8be276143a962dcfeb5ea514d50d656fe8d4e5c15d6df5c2e0R16 

### I expected this:

_No response_

### Is there a workaround?

_No response_

### Anything else?

_No response_

### EventCatalog Version

2.19.0

### Node.js Version

22

### Platform(s)

Linux

### Community Notes

<!-- Please keep this note for the community -->
* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

--- a/.changeset/beige-moles-hunt.md
+++ b/.changeset/beige-moles-hunt.md
@@ -1,0 +1,5 @@
+---
+"@eventcatalog/core": patch
+---
+
+fix(core): ubiquitous language pages now render without any data

--- a/eventcatalog/src/pages/docs/[type]/[id]/language/[dictionaryId]/index.astro
+++ b/eventcatalog/src/pages/docs/[type]/[id]/language/[dictionaryId]/index.astro
@@ -2,7 +2,7 @@
 import RectangleGroupIcon from '@heroicons/react/24/outline/RectangleGroupIcon';
 import Footer from '@layouts/Footer.astro';
 import VerticalSideBarLayout from '@layouts/VerticalSideBarLayout.astro';
-import { getDomains, type Domain, getUbiquitousLanguage } from '@utils/collections/domains';
+import { getDomains, getUbiquitousLanguage } from '@utils/collections/domains';
 import { buildUrl } from '@utils/url-builder';
 import { ClientRouter } from 'astro:transitions';
 import { marked } from 'marked';
@@ -13,6 +13,11 @@ export async function getStaticPaths() {
   const pages = await domains.reduce<Promise<any[]>>(async (acc, domain) => {
     const accumulator = await acc;
     const ubiquitousLanguages = await getUbiquitousLanguage(domain);
+
+    if (ubiquitousLanguages.length === 0) {
+      return accumulator;
+    }
+
     const dictionary = ubiquitousLanguages[0].data.dictionary;
 
     if (!dictionary) {

